src/core/pointcloud/qgspointcloudrenderer.cpp

```diff
@@ -255,8 +255,8 @@ void QgsPointCloudRenderer::saveCommonProperties( QDomElement &element, const Qg
 
   element.setAttribute( QStringLiteral( "maximumScreenError" ), qgsDoubleToString( mMaximumScreenError ) );
   element.setAttribute( QStringLiteral( "maximumScreenErrorUnit" ), QgsUnitTypes::encodeUnit( mMaximumScreenErrorUnit ) );
-  element.setAttribute( QStringLiteral( "pointSymbol" ), qgsEnumValueToKey( mPointSymbol ) );
-  element.setAttribute( QStringLiteral( "drawOrder2d" ), qgsEnumValueToKey( mDrawOrder2d ) );
+  element.setAttribute( QStringLiteral( "pointSymbol" ), QString::number( static_cast< int >( mPointSymbol ) ) );
```

Comment:
maybe a little comment here so nobody would make the same mistake trying to fix things

Reply:
There's literally 100s (maybe 1000s) of places in the code we do this. I don't see the need to add a comment for just this one random occurrence. 

It's just something we reviewers need to be aware of as a general principle and catch before merges.